### ISSUE TITLE
Feature request: limiting bandwidth for contributed anonymity traffic

### ISSUE THREAD

**Issue by vi** [Tribler/tribler#2187]
I see "Total:" growing and "Balance:" falling a bit rapidly.

Sometimes I want to limit (but not turn off completely) bandwidth contributed to decentralized networks.

Ideally there should be easily switchable "Traffic profiles" in Tribler affecting any network activity (including zero - temporarily stop Tribler from using network, just to view downloads, edit settings, create torrents for future upload and so on).

**Comment by ichorid** [comment]
@egbertbouman , can we add some kind of rate control to the Tunnels ? How hard would that be?

**Comment by egbertbouman** [comment]
@ichorid The anonymous downloader and seeder fall under the normal rate control. That leaves rate control for relaying, which I think we shouldn't focus on at this point. At most, we could allow users to turn of relaying.

**Comment by ichorid** [comment]
I agree that we should at least provide an option to turn off relaying. The more efficient Tunnels get, the more important this becomes.

**Comment by phoglenix** [comment]
Maybe related to the recent increased need to use more anonymous hops: sometimes I'm seeing tribler close to saturating my internet connection with traffic that isn't listed as upload or download traffic. I assume this is intermediate node relaying traffic. There's no bandwidth limit for this so I'm forced to just kill the application if it uses too much when I need the bandwidth. Please consider prioritising a bandwidth limit setting for relaying traffic.
